2018-02-24-Official-Mapping-Contests:-osu!-Mapping-Olympiad.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post 
+title: "Official Mapping Contests: osu! Mapping Olympiad"
+date: 2018-02-24
+--- 
+
+The osu! Mapping Olympiad marks the return of official beatmapping contests. Starting with the osu! gamemode, we're bringing back beatmapping contests in force! Introducing the first contest: Pick one of three songs for your chance to become osu!'s next *Elite Mapper*!
```

Comment:
seems to conflict with line 62, did you mean "Mapping Olympian" instead of "Elite Mapper"?